openstack/big_patch.py

```diff
@@ -668,6 +668,22 @@ def deploy_to_node(self, node, nodes_information):
         self.check_lldpd_running(node)
         self.check_bond_int_speeds_match(node, bond_interfaces)
 
+        # copy neutron metadata file to all nodes
+        resp, errors = self.env.copy_file_to_node(node, "/etc/neutron/metadata_agent.ini",
+                                                  "/etc/neutron/metadata_agent.ini")
+        if errors:
+            raise Exception("error pushing updated metadata.ini to %s:\n%s"
```

Comment:
Don't raise exceptions. Instead, stop metadata agent and dhcp agent on that node